types/schema.d.ts

```diff
@@ -2,7 +2,7 @@
  * Schemas in Fastify follow the JSON-Schema standard. For this reason
  * we have opted to not ship strict schema based types. Instead we provide
  * an example in our documentation on how to solve this problem. Check it
- * out here:
+ * out here: https://github.com/fastify/fastify/blob/main/docs/Validation-and-Serialization.md
```

Comment:
Should not be the https://github.com/fastify/fastify/blob/main/docs/TypeScript.md#json-schema link?

Reply:
Thanks, hadn't see there was a direct example, fixed